bridge/client/app/_views/ktb-settings-view/ktb-integration-view/ktb-integration-view.component.ts

```diff
@@ -1,158 +1,102 @@
-import { Component, OnDestroy, OnInit, TemplateRef } from '@angular/core';
+import { Component, TemplateRef } from '@angular/core';
 import { DtSortEvent, DtTableDataSource } from '@dynatrace/barista-components/table';
-import { BehaviorSubject, combineLatest, Observable, Subject } from 'rxjs';
+import { combineLatestWith, merge, Observable, shareReplay, Subject, switchMap } from 'rxjs';
 import { DataService } from '../../../_services/data.service';
 import { ActivatedRoute, Router } from '@angular/router';
-import { filter, map, switchMap, takeUntil } from 'rxjs/operators';
+import { distinctUntilChanged, filter, finalize, map, tap } from 'rxjs/operators';
 import { UniformRegistrationLog } from '../../../../../shared/interfaces/uniform-registration-log';
 import { UniformRegistration } from '../../../_models/uniform-registration';
 import { Location } from '@angular/common';
-import { UniformSubscription } from '../../../_models/uniform-subscription';
+
+export type Params = { projectName: string; integrationId?: string };
+
+const sortConfig: Record<string, (u: UniformRegistration) => string> = {
+  host: (u) => u.metadata.hostname,
+  namespace: (u) => u.metadata.kubernetesmetadata.namespace,
+  location: (u) => u.metadata.location,
+};
+
+const empty = '--EMPTY--';
```

Comment:
I guess this is a workaround for the `ngIf`. It can be removed if we use `{value: myValue | async}` instead.

Reply:
Fixed